R/module_teal.R

```diff
@@ -197,5 +204,9 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
     mapping_table <- srv_filter_manager_panel("filter_manager_panel", slices_global = slices_global)
     snapshots <- srv_snapshot_manager_panel("snapshot_manager_panel", slices_global = slices_global)
     srv_bookmark_panel("bookmark_manager", modules)
+
+    if (inherits(data, "teal_data_module")) {
+      setBookmarkExclude(c("teal_modules-active_tab"))
+    }
```

Comment:
Wrong namespace. I think exclusion should be set in `srv_teal_module.teal_modules`

```suggestion
```

------

Besides, I think we can do something like this. WDYT? If you agree it is a good idea please add todo comment in `srv_teal_module.teal_modules` (this is where active tab exists

```r
# pseudocode
onBookmark({
  if (input$active_tab == "teal_data_module") {
    input_list[[-active_tab]]
  }
})

```